Tests/LibWeb/Text/expected/wpt-import/css/css-pseudo/file-selector-button-inherit.txt

```diff
@@ -0,0 +1,11 @@
+Summary
+
+Harness status: OK
+
+Rerun
+
+Found 1 tests
+
+1 Pass
+Details
+Result	Test Name	MessagePass	::file-selector-button should inherit from its originating element	
```

Comment:
This test needs to be rebaselined, as the expectation file format for imported WPT tests changed in #2714.